package.json

```
@@ -13,14 +13,14 @@
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
         "watch:sass": "sass --watch src/assets/scss:src/assets/css",
         "watch:eleventy": "eleventy --serve --port=2022",
-        "build:sass": "sass src/assets/scss:src/assets/css",
+        "build:sass": "sass --no-source-map --style=compressed src/assets/scss:src/assets/css",
         "build:eleventy": "npx @11ty/eleventy",
         "fetch:team": "node tools/fetch-team-data.js",
         "fetch:stats": "node tools/fetch-stats.js",
         "fetch:sponsors": "node tools/fetch-sponsors.js",
         "fetch": "npm-run-all --parallel fetch:*",
-        "start": "npm-run-all build:sass --parallel watch:*  --parallel images",
-        "build": "npm-run-all build:sass --parallel build:*   --parallel images"
+        "start": "npm-run-all build:sass --parallel watch:* images",
+        "build": "npm-run-all build:sass --parallel build:eleventy images"
```

```suggestion
        "build": "npm-run-all build:sass build:eleventy images"
```

None of these 3 tasks can be performed in parallel with any other because each of them operates on the result of the previous task.

`build:eleventy` must wait for `build:sass` to compile `src/assets/scss/*` to `src/assets/css/*`

`images` must wait for `build:eleventy` to copy images to `_site/assets/images/`